Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```
@@ -58,6 +58,7 @@ span.DropDownMenuSelect a.DropDownMenuLabel
   padding: var(--remotion-themed-spacing);
   border-right: var(--remotion-themed-border-width) solid var(--color-contrast-separator);
   outline: none;
+  font-family: var(--remotion-themed-font-bold);
```

The full specifier will not work for `font-family` so we need to use `font` here. There are also a few other places where we need to make this change.

```suggestion
  font: var(--remotion-themed-font-bold);
```